app/bundles/LeadBundle/Form/Type/ConfigType.php

```diff
@@ -25,14 +27,49 @@ class ConfigType extends AbstractType
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->add('background_import_if_more_rows_than', 'number', [
-            'label'      => 'mautic.lead.background.import.if.more.rows.than',
-            'label_attr' => ['class' => 'control-label'],
-            'attr'       => [
-                'class'   => 'form-control',
-                'tooltip' => 'mautic.lead.background.import.if.more.rows.than.tooltip',
-            ],
-        ]);
+        $builder->add(
+            'background_import_if_more_rows_than',
+            NumberType::class,
+            [
+                'label'      => 'mautic.lead.background.import.if.more.rows.than',
+                'label_attr' => ['class' => 'control-label'],
+                'attr'       => [
+                    'class'   => 'form-control',
+                    'tooltip' => 'mautic.lead.background.import.if.more.rows.than.tooltip',
+                ],
+            ]
+        );
+
+        $builder->add(
+            'import_max_runtime',
+            NumberType::class,
+            [
+                'label'      => 'mautic.lead.import.max.runtime',
+                'label_attr' => ['class' => 'control-label'],
+                'attr'       => [
+                    'class'   => 'form-control',
+                    'tooltip' => 'mautic.lead.import.max.runtime.tooltip',
+                ],
+            ]
+        );
+
+        $statusChoices = [];
+        foreach ([7, 4, 5] as $validStatus) {
```

Comment:
We should use constants here, instead of integers, so that we know what these are.

Reply:
Done